lineman/app/services/ability_service.coffee

```diff
@@ -8,9 +8,9 @@ angular.module('loomioApp').factory 'AbilityService', (CurrentUser) ->
 
     canEditThread: (thread) ->
       CurrentUser.isMemberOf(thread.group()) and
-      (@canAdministerGroup(thread.group()) or
-      CurrentUser.isAuthorOf(thread) or
-      thread.group().membersCanEditDiscussions)
+        (@canAdministerGroup(thread.group()) or
```

Comment:
This is the same, isn't it? Is there a fix in here for thread editing permissions?